### ISSUE TITLE
// @ts-ignore cannot suppress 'cannot resolve symbol'

### ISSUE THREAD

**Issue by franksongca** [palantir/tslint#4616]
### Bug Report

- __TSLint version__:5.12.1
- __TypeScript version__:3.3.3
- __Running TSLint via__: WebStorm

#### TypeScript code being linted

```ts
// code snippet
// @ts-ignore
import { AuxConst } from '../../../common/const.js';


It complains: cannot resolve symbol 'AuxConst'. I would need to ignore this error, it was working before.

**Comment by adidahiya** [comment]
That's not a tslint error, it looks like the typescript compiler `tsc`
